### PR TITLE
Add the option to include devcontainer

### PR DESCRIPTION
**Pull request by xanderificnl**
As per comments in https://github.com/laravel/sail/pull/203

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -22,9 +22,11 @@
 
     $services = $request->query('with', 'mysql,redis,meilisearch,mailhog,selenium');
 
+    $devcontainer = ($request->has('devcontainer') ? "--devcontainer" : "");
```
Comment (driesvints) [inline routes/web.php:25]:
```suggestion
    $devcontainer = $request->has('devcontainer') ? '--devcontainer' : '';
```

**Comment by driesvints** [comment]
Thanks! 

@xanderificnl you'll also need to update the tests. Think it's just the extra space you added. Feel free to mark this as ready when that's done.

**Comment by xanderificnl** [comment]
Instead of adding the space, I actually removed the last quotation mark. It seemed to make more sense. In the future more things may be added and adding a space before each seems unneeded. Thoughts?

File: tests/Feature/SailServerTest.php [hunk 2]
```diff
@@ -22,4 +22,11 @@ public function test_different_services_can_be_picked()
         $response->assertStatus(200);
         $response->assertSee('php ./artisan sail:install --with=postgresql,redis,selenium');
     }
+
+    public function test_it_adds_the_devcontainer_upon_request() {
```
Comment (driesvints) [inline tests/Feature/SailServerTest.php:26]:
```suggestion
    public function test_it_adds_the_devcontainer_upon_request()
    {
```

**Comment by driesvints** [comment]
@xanderificnl The quotation mark actually shows that there are no unwanted extra params added so I'd undo that.

Can you also revert the changes to the composer.lock file?

**Comment by xanderificnl** [comment]
Yes, I'll update those things. I'm having a small fight on my hands with the composer file, I'll get it sorted. Thanks for reviewing so far!

**Comment by xanderificnl** [comment]
I think we nailed it!

**Comment by driesvints** [comment]
Thanks @xanderificnl

**Comment by taylorotwell** [comment]
Thanks for contributing to Laravel! ❤️

**Pull request merged**